docs/notebooks/tutorials/04_Omnigenity_Optimization.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>may also want to mention what subset of the parameters corresponds to QS</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>